src/view/web/components/money-input.js

```diff
@@ -0,0 +1,113 @@
+const moneyInputTemplate = document.createElement("template");
+moneyInputTemplate.innerHTML = `
```

Comment:
아래와 같은 Temlate Leteral 태그 기능을 사용하시면 IDE에서 코드 확인하기가 좋을거 같아요 ㅎㅎ

```suggestion
moneyInputTemplate.innerHTML = /*html*/`
```